### PR TITLE
Update Swate manual core features and delete installation guides

### PR DESCRIPTION
**Pull request by StellaEggels**
Deleted the Swate installation guides and updated the Core Features guides of the Swate Manual to Swate alpha.
@Freymaurer 
#403

### CODE REVIEW

File: src/docs/SwateManual/Docs03-Building-Blocks.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-date: 2022-09-19
+date: 2024-04-29
 author: 
 - name: Kevin Frey
   github: https://github.com/Freymaurer
   orcid: https://orcid.org/0000-0002-8510-6810
+- name: Stella Eggels
 add toc: false
 add sidebar: _sidebars\mainSidebar.md
 ---
```
[the remark this answers is not in the document]
Comment (Freymaurer) [reply src/docs/SwateManual/Docs03-Building-Blocks.md:12]: In addition i really like the styling in this image:
https://github.com/nfdi4plants/nfdi4plants.knowledgebase/blob/6c4aca9cb91dc0c9f62f3547fe28b5e9fd04347a/src/docs/img/Swate_a_overview2.png

Maybe we could use a unified styling for markers and arrows

File: src/docs/img/Swate_a_relatedtermsearch.png
Comment (Freymaurer) [inline src/docs/img/Swate_a_relatedtermsearch.png:1]: Again maybe use the styling from the overview image

**Review by Freymaurer — CHANGES_REQUESTED** [review]
Thanks a lot for your work @StellaEggels ! I really appreciate it!

**Comment by StellaEggels** [comment]
@Freymaurer I made the changes you suggested and changed the isa-json guide to WIP. If you agree, then please merge.

**Review by Freymaurer — CHANGES_REQUESTED** [review]
Thanks for the changes done! A few more things, then it is ready to merge 🙂

**Review by Freymaurer — APPROVED** [review]

**Pull request merged**